docs/deprecated/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,967 @@
+---
+sidebar_label: "Required Permissions"
+title: "Required Permissions"
+description: "Review the required permissions for deploying clusters on Azure"
+icon: ""
+hide_table_of_contents: false
+tags: ["public cloud", "azure", "permissions"]
+sidebar_position: 100
+---
+
+Palette requires a set of permissions to properly deploy and manage the lifecycle of clusters deployed to Azure. We
+recommend creating a
+[role assignment](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments) that has the
+service principal you want to use with Palette and the appropriate
+[scope level](https://learn.microsoft.com/en-us/azure/role-based-access-control/scope-overview). To ensure that Palette
+and VerteX can deploy and manage clusters on Azure in all use cases, use a subscription as the scope level for the role
+assignment.
+
+:::warning
+
+We recommend against assigning the built-in Azure
+[Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) role to the
+service principal you want to use in Palette because its permission scope exceeds our requirements. Instead, create a
+custom role to provide only those permissions that Palette requires.
+
+:::
+
+We support the following use cases:
+
+- [Dynamic Placement](#dynamic-placement) - Palette creates network resources required for your cluster.
+
+- [Static Placement](#static-placement) - Palette deploys clusters on the pre-existing network resources you specify.
+
+:::tip
+
+You can use [Validator](https://github.com/spectrocloud-labs/validator) with the
+[Azure plugin](https://github.com/spectrocloud-labs/validator-plugin-azure) to verify you have setup the correct
+permissions. The Validator Azure plugin requires the following permissions:
+
+- Microsoft.Authorization/denyAssignments/read
+- Microsoft.Authorization/roleAssignments/read
+- Microsoft.Authorization/roleDefinitions/read
+
+You can use Validator with the Palette CLI. Check out the Palette CLI
+[Validator](../../../automation/palette-cli/commands/ec.md#validate-environment) reference page for more information.
+
+:::
+
+Review the sections below to learn how to create a custom role in Azure and assign it to the service principal you want
+to use with Palette.
+
+## Dynamic Placement
+
+Dynamic placement is the default method for deploying Azure clusters through Palette. With dynamic placement, Palette
+dynamically creates the network resources required for your cluster.
+
+### Prerequisites
+
+- Azure CLI installed on your local machine. Refer to the
+  [Azure CLI Install Guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) for installation instructions.
+
+- The Azure subscription ID you want to use with Palette.
+
+- The Security Principal Object ID you want to use with Palette. You can retrieve it from the Entra ID section of the
```

Comment:
From an Azure perspective (whether you're doing Palette stuff or not), "security principal" is the most appropriate term. Validator uses this term too. But when it comes to using Palette, "service principal" is the most appropriate term because it's more specific and you wouldn't use a user (another type of security principal) to deploy a cluster using Palette.

Not going to leave another comment about this, we just need to do a find and replace on this page.

Reply:
Yup, this is the deprecated file. We just keep it for historical purposes 😄